resources/views/widgets/settings/alertlog.blade.php

```diff
@@ -22,4 +22,19 @@
             @endforeach
         </select>
     </div>
+    <div class="form-group row">
```

Comment:
form-group is a row, row is unneeded.

Reply:
just kept the style of file in consistence ...

Reply:
@murrant removed all "row" classes